web/src/types/common/index.ts

```diff
@@ -85,3 +85,8 @@ export interface KeyValueInfo {
   key: string;
   value: string;
 }
+
+export enum DivisionEnum {
+  CRIMINAL = 'R',
+  CIVIL = 'I'
+}
```

Comment:
It's best practice to avoid defining these sorts of lookup codes in the frontend.